Airports/EGHO/Basic.txt

```diff
@@ -1,3 +1,3 @@
 Thruxton
 N051.12.38.000 W001.36.00.000
-118.270
+118.275
```

Comment:
The AIP @ `EGHO AD 2.18` says that the A/G Operator's frequency is 118.280, and in our current .ESE that frequency isn't occupied?
```suggestion
118.280
```